pingcap/tidb
# metrics: remove fixed width in some legends

### What problem does this PR solve?


Issue Number: close #48153

Problem Summary:
Remove the fixed width.

### What is changed and how it works?

### Check List

Tests 

1. start a cluster
2. import the new json file
3. check TiDB -> CPS by instance
![image](https://github.com/pingcap/tidb/assets/4352397/c518a523-c96f-4993-87f8-639bdba1ec30)

  > - [ ] I checked and no code files have been changed.
  >   

Side effects


Documentation


### Release note



Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```
